utils.ts

```diff
@@ -241,3 +237,38 @@ export const mergeResponses = (
 
   return mergedResponse
 }
+
+/**
+ * Makes a geocoder request by first checking a potential redis store for a cached
+ * response. If a response is fetched, stores response in redis store.
+ * @param redisClient     Redis client already connected
+ * @param requestMethod   Geocoder Request Method
+ * @param args            Args for Geocoder request method
+ * @returns               FeatureCollection either from cache or live
+ */
+export const cachedGeocoderRequest = async (
+  geocoder: Record<string, (q: AnyGeocoderQuery) => Promise<FeatureCollection>>,
+  requestMethod: string,
+  args: AnyGeocoderQuery,
+  redisClient: RedisClientType | null
+): Promise<FeatureCollection> => {
+  const { focusPoint, text } = args
+  if (!text) return { features: [], type: 'FeatureCollection' }
+  const stringifiedFocusPoint = `${focusPoint?.lat}.${focusPoint?.lon}`
+  const redisKey = text + stringifiedFocusPoint
```

Comment:
FYI, if we use `:` as a separator, Medis will break it down into a tree view. Not sure if that realllly makes sense for this though.